content/en/docs/releases/next-release-preview/index.md

```diff
@@ -46,15 +46,19 @@ that this SpEL should be evaluated by a different tool. No exceptions are thrown
 ${#toJson(#doNotEval(fileMap))}
 ```
 
-Use a feature flag to enable.
+The feature flag introduced in Kork (1.28/29 releases) to use `doNotEval` SPeL expression helper will be enabled by default.
```

Comment:
A bit more text on what this feature does?  May be documented elsewhere but at a glance it's a touch "huh" if I didn't already know about it.